.github/workflows/release/build-images.yaml

```diff
@@ -0,0 +1,51 @@
+name: "[Release] Build service images"
+on: [push, pull_request]
+concurrency:
+    group: pulls/${{ github.event.pull_request.number || github.ref }}
+    cancel-in-progress: true
+jobs:
+    nango-jobs:
+        uses: ./.github/workflows/release/push-container.yaml
+        secrets: inherit
+        with:
+            package: jobs
+            run-cmd: ts-build
+            tags: -t nangohq/nango-jobs:${{ github.sha }}
+    nango-runner:
+        uses: ./.github/workflows/release/push-container.yaml
+        secrets: inherit
+        with:
+            package: runner
+            run-cmd: ts-build
+            tags: -t nangohq/nango-runner:${{ github.sha }}
+    nango-server-staging:
+        uses: ./.github/workflows/release/push-container.yaml
+        secrets: inherit
+        with:
+            package: server
+            run-cmd: build:staging
+            tags: -t nangohq/nango-server:staging-${{ github.sha }}
```

Comment:
I propose to use a single docker repo (`nangohq/nango-server`) for all server images with 4 tag tracks:
- production
- staging
- hosted
- enterprise

If you are ok with it. I will then modify the `nango-render-cloud` repo to use the images for prod and staging